README.md

```diff
@@ -480,6 +481,18 @@ When you want to get rid of all of your untracked files
 $ git clean -f
 ```
 
+<a href="I-want-to-unstage-specific-staged-file"></a>
+### I want to unstage specific staged file
+
+Sometimes we have one or more files that accidentally ended up being staged, and these files have not been committed before, to unstage them
+
+```sh
+$ git reset -- filename
+```
+
+this result in unstaging the file and make it look like it's untracked, then you can decide what to do with it, either delete or add a .gitignore entry.
```

Comment:
"This unstages the file, and makes it look like it's untracked."

No need for the second sentence. Sometimes we just want a file to be unstaged.

Reply:
Removed the 2nd sentence.